relay-general/src/protocol/contexts/device.rs

```diff
@@ -1,8 +1,27 @@
+use std::collections::HashMap;
+
+use once_cell::sync::Lazy;
+
 use crate::protocol::FromUserAgentInfo;
 use crate::store::user_agent::is_known;
 use crate::types::{Annotated, Object, Value};
 use crate::user_agent::{parse_device, ClientHints};
 
+/// A hashmap that translates from the android model to the more human-friendly product-names.
+/// E.g. NE2211 -> OnePlus 10 Pro 5G
+pub static ANDROID_MAP: Lazy<HashMap<&'static str, &'static str>> = Lazy::new(|| {
+    let mut map = HashMap::new();
+    let android_str = include_str!("../../../../files/android_models.csv");
```

Comment:
From the [docs](https://doc.rust-lang.org/std/macro.include_str.html):

> So, for instance, an invocation with a Windows path containing backslashes \ would not compile correctly on Unix.

I think we should add a note about that.

Reply:
Yes, and a contributor too for example. I'm not sure how many people are in that situation so my ask is not to do a fancy solution for them.

Reply:
yep yep